lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/oauth/OAuthClient.json

```diff
@@ -30,7 +30,7 @@
 		},
 		"verified": {
 			"type": "boolean",
-			"description": "initially false, becomes true once the client creator demonstrates ownership of the redirect URI host",
+			"description": "initially false, to be verified please contact the AC Team.",
```

Comment:
Maybe add link to https://docs.synapse.org/articles/using_synapse_as_an_oauth_server.html